script.js

```diff
@@ -0,0 +1,61 @@
+$(document).ready(function() {
+	console.log("Got here!");
+
+	/*
+	1. Find the update button
+	2. Add a click handler to the update button
+	*/
+
+	$("#submit-btn").click(changePhoto);
+
+	function changePhoto () {
+
+		event.preventDefault();
+
+	//3. Find the user input and store in var city
+
+		var city = $("#city-type").val();
+
+	/*
+	4. If placeholder = "New York" "New York City" or "NYC" 
+	body background: url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/nyc.jpg)
+	
+	If placeholder = "San Francisco" "SF" or "Bay Area"
+	body background: url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/sf.jpg)
+
+	If placeholder = "Los Angeles" "LA" or "LAX"
+	body background: url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/la.jpg)
+
+	If placeholder = "Austin" or "ATX"
+	body background: url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/austin.jpg)
+
+	If placeholder = "Sydney" or "SYD"
+	body background: url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/sydney.jpg)
+	*/
+
+		if (city === "New York City" || city === "New York" || city === "NYC") {
+			$("body").css("background-image", "url(file:///Users/christinarust/Desktop/FEWD31/Homework/Assignment5/images/nyc.jpg)");
```

Comment:
Ahhh! very interesting approach. This would totally get the job done, except for one minor problem: the url's you've included are tightly coupled to *your* machine. You see how it says `file:///Users/christinarust/`? This will work great but only if you test it on your computer. The minute you try to deploy this to a real website, it won't work. 

You could address it by using relative paths, e.g.:
```js
$("body").css("background-image", "url(images/nyc.jpg)");
```
Now again, you took a very unique approach to solving this problem. Rather than changing out pre-existing classes (https://github.com/yirklum/FEWD31-Assignment5/blob/master/css/style.css#L126-L140), you just inlined new CSS to force changing the background image. Again your approach totally works. I would invite you to take a look at the solution I came up with. It involves just adding and removing CSS classes that were already defined in the stylesheet

My solution:
https://github.com/ericmasiello/FEWD31-Assignment5/blob/solution/main.js